kuma/wiki/templatetags/ssr.py

```diff
@@ -134,11 +134,7 @@ def server_side_render(component_name, data):
         result = response.json()
         return _render(component_name, result['html'], result['script'])
 
-    except requests.exceptions.ConnectionError:
-        print("Connection error contacting SSR server.")
-        print("Falling back to client side rendering.")
-        return client_side_render(component_name, data)
-    except requests.exceptions.ReadTimeout:
-        print("Timeout contacting SSR server.")
+    except requests.exceptions.RequestException as exception:
```

Comment:
A `ConnectionError` and a `ReadTimeout` exception are both subclasses of `RequestException`. So is `HTTPError` which would happen if the network talk to the URL worked but the server responded with a 500 error. 

This will NOT raise a Python exception:
```python
url = "https://httpbin.org/status/500"
r = requests.get(url)
```
But this will:
```python
url = "https://httpbin.org/status/500"
r = requests.get(url)
r.raise_for_status()
```
In fact, you get:
```
Traceback (most recent call last):
...
requests.exceptions.HTTPError: 500 Server Error: INTERNAL SERVER ERROR for url: https://httpbin.org/status/500
```